### PR TITLE
mom-2023-09-06

### PR DESCRIPTION
**Pull request by diegogonmar**
#### What type of PR is this?

* documentation

#### What this PR does / why we need it:

MoM of the call held in 2023-09-06.

#### Which issue(s) this PR fixes:

None

### CODE REVIEW

**Review by bigludo7 — APPROVED** [review]
Thanks Diego for your consideration of my requests for change.

File: documentation/MeetingMinutes/MOM-2023-09-06.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+| **Closed issues** |  |  |
+| [#15](https://github.com/camaraproject/IdentityAndConsentManagement/issues/15) | GSMA/Vonage | **Subscriber Identity API**<br> Was closed as agreed on 23/8 call as there were no further comments nor objections. |
+| [#52](https://github.com/camaraproject/IdentityAndConsentManagement/issues/52) ([PR#54](https://github.com/camaraproject/IdentityAndConsentManagement/pull/54)) | GSMA | **Differences in authentication on APIs between deployments can lead to differences in API usage and break federation**<br> Closed after merge of PR #54 |
+| **New issues** |  |  |
+| [#57](https://github.com/camaraproject/IdentityAndConsentManagement/issues/57) ([PR#33](https://github.com/camaraproject/IdentityAndConsentManagement/pull/33))| Telefonica | **Align securitySchemes and security of CAMARA API specs with IdentityAndConsentManagement WG**<br> - Will progress once Commonalities PR is merged. Discussion will continue offline. |
```
Comment (rartych) [inline documentation/MeetingMinutes/MOM-2023-09-06.md:84]: I am not sure if the issue is related to PR#33.
```suggestion
| [#57](https://github.com/camaraproject/IdentityAndConsentManagement/issues/57) ([PR#33](https://github.com/camaraproject/IdentityAndConsentManagement/pull/33))| Telefonica | **Align securitySchemes and security of CAMARA API specs with IdentityAndConsentManagement WG**<br> - Will progress once [Commonalities](https://github.com/camaraproject/Commonalities/pull/57) PR  is merged. Discussion will continue offline. |
```

Comment (diegogonmar) [reply documentation/MeetingMinutes/MOM-2023-09-06.md]: Yes, that was wrong. Actually that issue does not have a related PR yet. Removing that and updating the link to commonalities PR in this commit: https://github.com/camaraproject/IdentityAndConsentManagement/pull/60/commits/94c9f9ea9a91294f70b985d2fdbb69239018e46a

**Review by rartych — APPROVED** [review]
LGTM

**Pull request merged**